### PR TITLE
Docs: Add Tutorial on How to Setup Hedera Local Node using NPM CLI

### PR DESCRIPTION
**Pull request by owans**
**Description**:

* Added Tutorial on How to Setup Hedera Local Node using NPM CLI

**Related issue(s)**:
* NIL

**Notes for reviewer**:
* NIL

**Checklist**
- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: tutorials/local-node/setup-hedera-node-cli-npm.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+# How to Set Up a Hedera Local Node using the NPM CLI Tool
+
+Hedera is an open-source, public, proof-of-stake network. Its network services offer low & fixed fees,10k TPS, and instant transaction finality. Learn more about the [Hedera platform and how it works](https://hedera.com/how-it-works).
```
Comment (theekrystallee) [inline tutorials/local-node/setup-hedera-node-cli-npm.md]: 1. we like to spell out the "&" typically. suggest: "low and fixed"
2. add a space between the first "," and "10k"

File: tutorials/local-node/setup-hedera-node-cli-npm.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+In this tutorial, we will adopt, set up, and run a Hedera node locally using the [@hashgraph/hedera-local](https://www.npmjs.com/package/@hashgraph/hedera-local) NPM Command Line Interface (CLI) tool with docker compose. 
+
+> This tutorial is based on the [Hedera Local Node README documentation](https://github.com/hashgraph/hedera-local-node).
+
+> Already familiar with using a cloud service? Check  out the other options for setting up and running the Hedera node locally. See the [Useful resources section](https://docs.google.com/document/d/1gWKWF-fzc0VlKhRhjZhecatHnTXkHy9RQeKfD6Klnak/edit#heading=h.5zlu1j5vb4rk) for more information.
+
+## Prerequisites
+
+To get started with this tutorial, ensure that you have the following software installed:
+* [Node.js](http://Node.js) >= v14.x (Check version: node -v)
```
Comment (theekrystallee) [inline tutorials/local-node/setup-hedera-node-cli-npm.md]: 1. replace the nodejs link http://node.js link to https://nodejs.org/
2. we like to highlight all commands that will be run in the terminal. please include the "``" backtick around the `node -v` command

File: tutorials/local-node/setup-hedera-node-cli-npm.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+* NPM >= v6.14.17 (Check version: npm -v)
+* [Docker](https://www.docker.com/) >= v20.10.x (Check version: docker -v)
+* [Docker Compose](https://docs.docker.com/compose/) >= v2.12.2 (Check version: docker compose version)
```
Comment (theekrystallee) [inline tutorials/local-node/setup-hedera-node-cli-npm.md]: same comment as the last of to include "``" around the terminal commands

`npm -v`
`docker -v`
`docker compose version`

File: tutorials/local-node/setup-hedera-node-cli-npm.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+* Hardware: Minimum 16GB RAM
+
+### Installation
+* Node.js and NPM: Refer to the [official installation guide](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs).
+* Docker: See [Docker Setup Guide](https://github.com/hashgraph/hedera-local-node?tab=readme-ov-file#note) to get docker up and running (note: specific instructions may vary based on the OS).
+
+## Getting Started
+
+Clone the Github repo, navigate to the project folder using the commands below;
```
Comment (theekrystallee) [inline tutorials/local-node/setup-hedera-node-cli-npm.md]: GitHub has a capital H

**Review by theekrystallee — CHANGES_REQUESTED** [review]
Thanks so much for the PR! This looks great. Just resolve a few of my comments and it should be ready to merge.

File: tutorials/local-node/setup-hedera-node-cli-npm.md (hunk 1 above)
Comment (owans) [reply tutorials/local-node/setup-hedera-node-cli-npm.md]: Noted. Thank you

File: tutorials/local-node/setup-hedera-node-cli-npm.md (hunk 3 above)
Comment (owans) [reply tutorials/local-node/setup-hedera-node-cli-npm.md]: Thank you

File: tutorials/local-node/setup-hedera-node-cli-npm.md (hunk 4 above)
Comment (owans) [reply tutorials/local-node/setup-hedera-node-cli-npm.md]: Good spot, will resolve

**Comment by owans** [comment]
Fixes resolved. Thank you for the feedback @theekrystallee.

**Review by theekrystallee — APPROVED** [review]
Thanks so much for making the changes that we asked for and for the additional points of clarity. This looks great!

**Pull request merged**